jina/executors/indexers/vector.py

```diff
@@ -26,25 +26,26 @@ class BaseNumpyIndexer(BaseVectorIndexer):
         Setting :attr:`compress_level`>0 gives a smaller file size on the disk in the index time. However, in the query
         time it loads all data into memory at once. Not ideal for large scale application.
 
-        Setting :attr:`compress_level`=0 enables :func:`np.memmap`, which loads data in an on-demanding way and
+        Setting :attr:`compress_level`=0 enables :func:`np.memmap`, which loads data in an on-demand way and
         gives smaller memory footprint in the query time. However, it often gives larger file size on the disk.
+
+    :param compress_level: The compresslevel argument is an integer from 0 to 9 controlling the
+                    level of compression; 1 is fastest and produces the least compression,
+                    and 9 is slowest and produces the most compression. 0 is no compression
+                    at all. The default is 9.
+    :param ref_indexer: Bootstrap the current indexer from a ``ref_indexer``. This enables user to switch
+                        the query algorithm at the query time.
     """
 
     def __init__(self,
                  compress_level: int = 1,
                  ref_indexer: Optional['BaseNumpyIndexer'] = None,
+                 delete_on_save=False,
```

Comment:
I think `delete_on_dump` may be more accurate, since `saving` the file happens more often in a Pea lifetime.

Reply:
Makes sense.